packages/amplify-category-auth/resources/lambda-function.js

```diff
@@ -12,7 +12,7 @@ exports.handler = function (event, context) {
       const authParams = { UserPoolId: userPoolId, LambdaConfig: {} };
       const cognitoclient = new aws.CognitoIdentityServiceProvider();
       promises.push(cognitoclient.updateUserPool(authParams).promise());
-      Promise.all(promises)
+      (async () => await Promise.all(promises))()
```

Comment:
this still leaves an unawaited promise hanging around. You need to make the function async and await the promise (it doesn't need to be wrapped in another function)

Reply:
done!